mantle/cmd/kola/qemuexec.go

```diff
@@ -63,6 +64,7 @@ func init() {
 	cmdQemuExec.Flags().StringSliceVar(&ignitionFragments, "add-ignition", nil, "Append well-known Ignition fragment: [\"autologin\"]")
 	cmdQemuExec.Flags().StringVarP(&hostname, "hostname", "", "", "Set hostname via DHCP")
 	cmdQemuExec.Flags().IntVarP(&memory, "memory", "m", 0, "Memory in MB")
+	cmdQemuExec.Flags().BoolVarP(&cpuCountHost, "auto-cpus", "C", false, "Automatically set number of cpus to host count")
```

Comment:
optional: I have a slight preference for not adding the short arg and just add it as a long argument.

Reply:
Hmm...yeah, OK.